components/camel-jira/src/main/java/org/apache/camel/component/jira/consumer/NewCommentsConsumer.java

```diff
@@ -60,21 +62,25 @@ protected int poll() throws Exception {
     }
 
     // In the end, we want *new* comments oldest to newest.
-    private Stack<Comment> getComments(boolean loadComments) {
-        Stack<Comment> newComments = new Stack<>();
+    @SuppressWarnings("ConstantConditions")
+    private List<Comment> getComments() {
+        LOG.info("Start: Jira NewCommentsConsumer: retrieving issue comments. Last comment id: {}", lastCommentId);
```

Comment:
Do we really need this INFO logging noise? If this method is invoked frequently you get spammed with logs. Can we lower this to DEBUG